New to git.md

```diff
@@ -63,77 +50,63 @@ few commands to be careful with.
+          `git reset --hard` as documented above.
 
-      * You might also be able to simply use `git rebase
-        origin/master` to remove the merge commit.
+      * You might also be able to simply use `git rebase origin/master` to remove the merge commit.
 
-    * If the merged commit is pushed upstream, then unfortunately it
-      will persist in history. This is not really a big deal, but just
-      be sure that the merge worked correctly. If you find that
-      changes were actually lost then add new commits to re-apply the
-      changes.
+    * If the merged commit is pushed upstream, then unfortunately it will persist in history. This is not really a big
+      deal, but just be sure that the merge worked correctly. If you find that changes were actually lost then add new
+      commits to re-apply the changes.
 
 * `git pull`
 
-  * By default, the `git pull` command is a `git fetch` followed by a
-    `git merge`. Therefore it has the same concerns as the `git merge`
-    command documented above.
+  * By default, the `git pull` command is a `git fetch` followed by a `git merge`. Therefore it has the same concerns
+    as the `git merge` command documented above.
 
   * Tip:
 
-    * You can also run `git config pull.rebase true` to set your edk2
-      tree up so that `git pull` will be a `git fetch` followed by a
-      `git rebase` (rather than `git merge`)
+    * You can also run `git config pull.rebase true` to set your edk2 tree up so that `git pull` will be a `git fetch`
+      followed by a `git rebase` (rather than `git merge`)
 
 * `git push -f`
 
-  * This command 'force' pushes a branch. This potentially can force
-    the remote branch to rewrite its history.
+  * This command 'force' pushes a branch. This potentially can force the remote branch to rewrite its history.
 
-  * Note: The main edk2 tree is protected from force pushes, and you
-    can setup your github branches to be similarly protected:
+  * Note: The main edk2 tree is protected from force pushes, and you can setup your github branches to be similarly
+    protected:
 
     <https://github.com/blog/2051-protected-branches-and-required-status-checks>
 
   * Why it is potentially dangerous:
 
-    * If you 'rewrite' the history of a tree, then people will be
-      suspicious that bad changes have been snuck into the tree.
+    * If you 'rewrite' the history of a tree, then people will be suspicious that bad changes have been snuck into the
+      tree.
 
-    * This is generally only considered bad for 'upstream' branches
-      that many people are basing their work off of.
+    * This is generally only considered bad for 'upstream' branches that many people are basing their work off of.
 
   * Ways to recover:
 
-    * You might be able to force push the old version back if you can
-      find out its version.
+    * You might be able to force push the old version back if you can find out its version.
 
   * It's sometimes okay to force push
```

Comment:
Leaving a comment made earlier by @mdkinney [here](https://github.com/makubacki/tianocore.github.io/commit/309c5ba227c0c4308c7955c56c84f39524db3441#r142054618):

> Do we want to mention that one of the workflows where forced push is allowed is when working on a branch in a fork that is a PR and the contributor wants to post a new version of a patch series with review feedback integrated.

Reply:
I will update to mention that.

Reply:
Done